hive-bigquery-connector-common/src/main/java/com/google/cloud/hive/bigquery/connector/output/OutputCommitterUtils.java

```diff
@@ -28,15 +28,12 @@
 public class OutputCommitterUtils {
 
   public static void commitJob(Configuration conf, JobDetails jobDetails) throws IOException {
-    try {
-      if (jobDetails.getWriteMethod().equals(HiveBigQueryConfig.WRITE_METHOD_DIRECT)) {
-        DirectOutputCommitter.commitJob(conf, jobDetails);
-      } else {
-        IndirectOutputCommitter.commitJob(conf, jobDetails);
-      }
-    } finally {
-      jobDetails.cleanUp(conf);
+    if (jobDetails.getWriteMethod().equals(HiveBigQueryConfig.WRITE_METHOD_DIRECT)) {
```

Comment:
the cleanup done by PostInsertHook is only in hive-1 module, what happens for cleanup in hive-2/3?